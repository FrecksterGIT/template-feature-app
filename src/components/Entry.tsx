import * as React from 'react';

export const Entry: React.FC = () => {
  return <h1>Demo-Entry #1</h1>;
};
