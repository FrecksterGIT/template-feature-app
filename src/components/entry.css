.headline {
  color: var(--color-red);
}
