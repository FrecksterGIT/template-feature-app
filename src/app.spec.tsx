/**
 * @jest-environment jsdom
 */
import * as React from 'react';

import * as TestRenderer from 'react-test-renderer';

import templateFeatureAppDefinition from './app';
import {Entry} from './components/entry';

test('output contains one entry point', () => {
  const featureApp = templateFeatureAppDefinition.create(undefined);
  const rendered = TestRenderer.create(<>{featureApp.render()}</>);

  const instance = rendered.root;
  expect(instance.findAllByType(Entry)).toHaveLength(1);
  expect(rendered.toJSON()).toMatchSnapshot();
});
