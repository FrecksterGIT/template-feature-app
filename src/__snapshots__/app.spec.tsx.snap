// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`output contains one entry point 1`] = `
<h3
  className="audi-headline-order-3 headline"
>
  Demo-Entry #1
</h3>
`;
